tests/v3_api/test_upgrade.py

```diff
@@ -221,28 +291,23 @@ def validate_service_discovery(sd_record_name, workload_names):
         validate_dns_record(pod, record, expected_ips)
 
 
-def create_wokloads_with_secret():
+def create_validate_wokloads_with_secret():
     value = base64.b64encode(b"valueall")
     keyvaluepair = {"testall": value.decode('utf-8')}
 
```

Comment:
Can you make this as a variable , that can be used in create_validate_wokloads_with_secret, validate_worklaods_with_secret and modify_resources_validate method ?

Reply:
keyvaluepair as a global variable to be used in different methods